src/modules/content-tree/components/list-item/list.item.component.js

```diff
@@ -78,21 +157,30 @@ class ListItem extends Component {
                     </a>
                 </div>
                 {children}
+                {this.renderLoadMoreBtn()}
```

Comment:
Why `listItem` has the load more button? it seems like the `list` component should have the load more button.

Reply:
When I first looked at it, for me it also seemed strange but it actually makes some sense. The List is container and ListItem is the real content (label, subitems, "load more" button).